packages/cascara/src/structures/AdminStructure/AdminStructure.js

```diff
@@ -30,9 +31,15 @@ const AdminStructure = ({ children, header, nav, theme = THEME_DEFAULTS }) => {
 
   return (
     <AdminProvider>
-      <header className={styles._header}>{header}</header>
-      <nav className={styles._nav}>{nav}</nav>
-      <AdminStructureChildren>{children}</AdminStructureChildren>
+      <header className={styles._header}>
+        <Boundaries>{header}</Boundaries>
+      </header>
+      <nav className={styles._nav}>
+        <Boundaries>{nav}</Boundaries>
+      </nav>
+      <AdminStructureChildren>
+        <Boundaries>{children}</Boundaries>
+      </AdminStructureChildren>
```

Comment:
Every slot on AdminStructure now has a wrapping boundary which will support both Suspense and error catching.